automated install/basic-install.sh

```diff
@@ -1505,19 +1505,24 @@ disable_dnsmasq() {
             echo -e "  ${TICK} Restarting Network manager"
             ${SUDO} systemctl reload-or-restart NetworkManager
         else
-            # Disabling dnsmasq via systemctl
-            echo -e "  ${INFO} Disabling dnsmasq via systemctl"
-            ${SUDO} systemctl disable dnsmasq &> /dev/null
+            # Disabling dnsmasq
+            disable_service dnsmasq
+            # Stopping dnsmasq
+            stop_service dnsmasq
         fi
         #setting dnsmasq_flag to false in order for the next check to validate or not
         dnsmasq_flag=false
         echo -e "  ${INFO} Checking if dnsmasq is still running"
         silent_port_53_check
         if [[ $dnsmasq_flag = "true" ]]; then
             echo -e "  ${EXCL} dnsmasq still active, this is most likely due to the fact that ${COL_LIGHT_RED}dnsmasq"
-            echo -e "    was loaded via a non convetional method. This might cause future conflicts with FTLDNS${COL_NC}"
-            ${SUDO} pkill dnsmasq
-            echo -e "  ${TICK} dnsmasq process killed"
+            echo -e "    was loaded via an init.d service.${COL_NC}"
+            # Stopping dnsmasq
+            echo -e "  ${TICK} Stopping dnsmasq via service command"
+            ${SUDO} service dnsmasq stop > /dev/null 2>&1
+            # Disabling dnsmasq
+            echo -e "  ${TICK} Disabling dnsmasq via update-rc.d command"
```

Comment:
These should be info instead of tick, since they haven't completed yet and are just info logs.